crosswalks/publiccode.csv

```diff
@@ -0,0 +1,74 @@
+Property,publiccode
+codeRepository,url
+programmingLanguage,
+review,
+runtimePlatform,platforms
+targetProduct,platforms
+applicationCategory,categories / description.[lang].genericName
+applicationSubCategory,categories
+downloadUrl,landingUrl
+fileSize,
+installUrl,landingUrl
```

Comment:
The landing URL is not machine-readable, so I think we should not map these properties to it.

Reply:
Do you mean the target is not readable or `landingUrl` itself is not machine readable? Because, if the latter, it should, since it's a URL :slightly_smiling_face:

Reply:
I mean the target